interfaces/builtin/opengl.go

```diff
@@ -114,6 +114,9 @@ unix (bind,listen) type=seqpacket addr="@cuda-uvmfd-[0-9a-f]*",
 # https://github.com/Xilinx/XRT/tree/master/src/runtime_src/core/edge/drm
 /sys/devices/platform/amba{,_pl@[0-9]*}/amba{,_pl@[0-9]*}:zyxclmm_drm/* r,
 
+# Imagination PowerVR driver
+/dev/pvr_sync rw,
```

Comment:
can you run `udevadm info --query property /dev/pvr_sync` and paste the output in this PR? it helps us ensure that the rule you provided for udev tagging is appropriate

Reply:
Sure. Here is the output of udevadm info
$ udevadm info --query property /dev/pvr_sync
DEVPATH=/devices/virtual/misc/pvr_sync
DEVNAME=/dev/pvr_sync
MAJOR=10
MINOR=61
SUBSYSTEM=misc
USEC_INITIALIZED=14533846
TAGS=:snap_ubuntu-core-kiosk_hook_connect-plug-graphics-core20:snap_ubuntu-core-kiosk_ubuntu-core-kiosk:snap_ubuntu-core-kiosk_hook_install:snap_ubuntu-core-kiosk_dae
mon:snap_ubuntu-core-kiosk_hook_post-refresh:snap_ubuntu-core-kiosk_hook_configure:

Reply:
thanks looks good